sympy/vector/deloperator.py

```diff
@@ -16,7 +16,7 @@ def __new__(cls, system=None):
                 feature="delop operator inside coordinate system",
                 useinstead="it as instance Del class",
                 deprecated_since_version="1.1",
-                issue=12858,
```

Comment:
There may have been some mixup here. This issue was supposed to say "vector" not "tensor". Since you already opened the other one with more information, should #12858 be closed?